### DISCUSSION TITLE
Error after modifing crazyswarm_server.cpp

### DISCUSSION THREAD

**Discussion by a240160572** [USC-ACTLab/crazyswarm#527] [Q&A]
I made modifications in src/crazyswarm_server.cpp, only on the logging frequency.

Now, I have the following error after I run the launch file (already sourced before):
`ERROR: cannot launch node of type [crazyswarm/crazyswarm_server]: Cannot locate node of type [crazyswarm_server] in package [crazyswarm]. Make sure file exists in package path and permission is set to executable (chmod +x)
ERROR: cannot launch node of type [crazyswarm/crazyswarm_teleop]: Cannot locate node of type [crazyswarm_teleop] in package [crazyswarm]. Make sure file exists in package path and permission is set to executable (chmod +x)`

Both files are executable from cmake file:
`add_executable(crazyswarm_server
  src/crazyswarm_server.cpp
)
target_link_libraries(crazyswarm_server
  ${catkin_LIBRARIES}
)

add_executable(crazyswarm_teleop
  src/crazyswarm_teleop.cpp
)
target_link_libraries(crazyswarm_teleop
  ${catkin_LIBRARIES}
)`

I tried to re-make the crazyswarm package but the make fails
`CMake Error at /opt/ros/melodic/share/catkin/cmake/assert.cmake:20 (message):
  

  Assertion failed: file
  '/home/lin/new_flip/crazy_swarm/ros_ws/src/crazyswarm/srv/crazyswarm.srv'
  does not exist.  Message: service file not found

Call Stack (most recent call first):
  /opt/ros/melodic/share/genmsg/cmake/genmsg-extras.cmake:170 (assert_file_exists)
  crazyswarm/CMakeLists.txt:54 (add_service_files)

-- Configuring incomplete, errors occurred!
See also "/home/lin/new_flip/crazy_swarm/ros_ws/build/CMakeFiles/CMakeOutput.log".
See also "/home/lin/new_flip/crazy_swarm/ros_ws/build/CMakeFiles/CMakeError.log".
Invoking "cmake" failed`

Can anyone help with that or what is the correct way to modify the src? Thank you

**Comment by whoenig** [comment]
1. Modify the source
2. Run `catkin_make` in `ros_ws`
3. Fix any build erros
